pygeoapi/util.py

```diff
@@ -519,7 +534,7 @@ def url_join(*parts: list) -> str:
     :returns: str of resulting URL
     """
 
-    return '/'.join([p.strip().strip('/') for p in parts])
+    return '/'.join([p.strip().strip('/') for p in parts]).rstrip('/')
```

Comment:
While writing and testing `get_base_url()` above, I noticed that this function returned paths with trailing slashes if you are concatenating empty parts or parts that strip down to nothing.
The `rstrip('/')` should prevent this.